docs/quickstart.rst

```diff
@@ -1,16 +1,31 @@
 Quick Start
 ===========
 
-The Cosmology API is a collection of runtime-checkable Protocols that describe
-the parts of the interface of a cosmology object. The Protocols can be used to
-describe and build functions that work with any compatible cosmology library
-(and any Array libraries that cosmology library might support), without even
-having the cosmology library as a hard dependency.
+The Cosmology API is a collection of runtime-checkable Protocols that define the
+interface of a cosmology object. The Protocols can be used to describe and build
+functions that work with any compatible cosmology library (and any Array
+libraries that cosmology library might support), without even having a single
+cosmology library as a run-time dependency.
+
+If you didn't understand the previous paragraph, don't worry, the previous
+section -- :doc:`doc introducing Python's typing and Protocols </introduction>`
+-- explains the concepts in more detail. Alternatively, we hope the examples in
+this Quick Start are sufficient you can just jump right in.
```

Comment:
I understand "section" to be a heading on a page, so I was confused for a second whether the "previous section" might have been accidentally deleted

```suggestion
If you didn't understand the previous paragraph, don't worry, the
:doc:`Introduction to Python Typing and Protocols </introduction>` explains the
concepts in more detail. Alternatively, we hope the examples in this Quick
Start are sufficient that you can just jump right in.
```

Reply:
I'l rephrase.